apps/portals/cancercomplexity/src/config/routesConfig.tsx

```diff
@@ -60,6 +68,13 @@ const routes: RouteObject[] = [
                 <ConsortiaGoals />
               </SectionLayout>
             </div>
+            <div className={'home-bg-dark'}>
+              <p className={'title center-title'}>
+                What Resources Are Available
+              </p>
+              <CardDeck {...availableResearchCardDeckConfig} />
```

Comment:
Prefer to use the `SectionLayout` component for rendering the title (handles linking to the section, ensures all titles on the home page have the right heading level)
```suggestion
              <SectionLayout title="What Resources Are Available">
                <CardDeck {...availableResearchCardDeckConfig} />
              </SectionLayout>
```

Reply:
Could you also update the `navbarConfig.ts` file in the CCKP project to add a link to this section under the 'Home' `children` property? Each of those entries creates a link in the Navbar that should take you straight to this section (you might want to check each in case another dev has forgotten to update this file)

![image](https://github.com/user-attachments/assets/eee0c37e-24c5-4827-99b4-35972e953fa7)